bindings/cs/MongoDB.Libmongocrypt/LibraryLoader.cs

```diff
@@ -168,6 +168,19 @@ private class LinuxLibrary : ISharedLibraryLoader
             // #define RTLD_GLOBAL     0x100
             public const int RTLD_GLOBAL = 0x100;
             public const int RTLD_NOW = 0x2;
+            private static readonly bool _use_libdl1;
+
+            static LinuxLibrary()
+            {
+                try
+                {
+                    Libdl1.dlerror();
+                    _use_libdl1 = true;
+                }
+                catch
+                {
```

Comment:
minor: I'd add `_use_libdl1 = false` in catch, just to readability (also technically it's better to initialize the variable)

Reply:
Done